src/assessor_tools.yml

```diff
@@ -47,6 +47,20 @@
           - requests
         unarchive_extra_opts:
           - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/trustedsec/CS-Remote-OPs-BOF\
+        /tarball/main"
+        install_dir: /tools/CS-Remote-Ops-BOF
+        unarchive_extra_opts:
+          - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: "https://github.com/trustedsec/CS-Situational-Awareness-BOF\
+        /tarball/master"
+        install_dir: /tools/CS-Situational-Awareness-BOF
+        unarchive_extra_opts:
+          - --strip-components=1
```

Comment:
Please also update the PR description to include the list of the tools that are being added.  That will be helpful to have here for posterity (you can just grab it from #140).

Reply:
Updated the PR description per @dav3r request.

Reply:
Thanks!

Reply:
I noticed that this conversation was resolved by @m1j09830, not @dav3r.  The person who made the initial comment should be the one to resolve it (unless you simply accept a set of proposed changes that the original reporter made).  Only the initial reporter can determine when the issue that prompted the comment has been resolved to his or her satisfaction.